### ISSUE TITLE
`image_processor` is missing in `get_mimicit_dataset`

### ISSUE THREAD

**Issue by ElegantLin** [Luodian/Otter#179]
Before you open an issue, please check if a similar issue already exists or has been closed before.

### When you open an issue, please be sure to include the following

- [ ] A descriptive title: Image Processor is missing in [`get_mimicit_dataset`](https://github.com/Luodian/Otter/blob/main/pipeline/train/data.py#L662C19-L662C19)
- [ ] A detailed description
- [ ] Assign an issue type tag (label):
  - `dataset` (mimic-it download, usage, etc.),
  - `demo` (online demo), `doc` (readme, wiki, paper, video etc.),
  - `evaluation` (evaluation result, performance of Otter etc.),
  - `model` (model configuration, components, etc.),
  - `train` (training configuration, process, code, etc.)

`Image_processor` is missing in [`get_mimicit_dataset`](https://github.com/Luodian/Otter/blob/main/pipeline/train/data.py#L607) function.

**Comment by Luodian** [comment]
Thanks, it's a bug during we updating our code. Let me fix it. Thanks.

**Comment by ElegantLin** [comment]
Thanks!

**Comment by baibizhe** [comment]
But [get_mimicit_dataset](https://github.com/Luodian/Otter/blob/main/pipeline/train/data.py#L607) haven't used the image processor . Is that right?

**Comment by Luodian** [comment]
Yes, it's used for webdataset loader for `MMC4` and `LAION2B`. It's used in OpenFlamingo's pretraining, as in Otter, we leave it as a placeholder to make the function interface compatible.

**Comment by baibizhe** [comment]
Thanks

**Comment by baibizhe** [comment]
So.If mimict_dataset does not use clip image processor.  However , in [otter_video.py](https://github.com/Luodian/Otter/blob/main/pipeline/demo/otter_video.py)  .It do use a clip image processor.  Why is this?
